week1/scary/lukvol/Main.hs

```diff
@@ -0,0 +1,16 @@
+import Data.Char
+import Data.List
+
+scareFactor :: Char -> Int
+scareFactor c 
+    | isLetter c = ord (toUpper c) - ord 'A' + 1
```

Comment:
`isLetter` would accept letters like `щ` as well (and you can't handle them). It's okay in this case, since the words file doesn't contain anything weird (I think, though I'm not sure whether it has anything with umlauts or not) – still, something to be aware of.